lib/strerror.c

```diff
@@ -308,6 +308,9 @@ curl_easy_strerror(CURLcode error)
   case CURLE_HTTP2_STREAM:
     return "Stream error in the HTTP/2 framing layer";
 
+  case CURLE_MISSING_TLS_CLOSE_NOTIFY:
+     return "Server closed the TLS connection without a close notification";
```

Comment:
Looks like there is a space indent too much in this line, please fix the indentation.